shared/common-adapters/index.js.flow

```diff
@@ -33,3 +33,43 @@ export {default as UserBio} from './user-bio'
 export {default as UserCard} from './user-card'
 export {default as UserProofs} from './user-proofs'
 export {default as Usernames} from './usernames'
+
+const NativeActivityIndicator: Object = {}
+const NativeAnimated: Object = {}
+const NativeClipboard: Object = {}
+const NativeEasing: Object = {}
+const NativeImage: Object = {}
+const NativeLinking: Object = {}
+const NativeListView: Object = {}
+const NativeModal: Object = {}
+const NativeNavigator: Object = {}
+const NativePicker: Object = {}
+const NativeScrollView: Object = {}
+const NativeStyleSheet: Object = {}
+const NativeSwitch: Object = {}
+const NativeText: Object = {}
+const NativeTextInput: Object = {}
+const NativeTouchableHighlight: Object = {}
+const NativeTouchableOpacity: Object = {}
+const NativeTouchableWithoutFeedback: Object = {}
+
+export {
```

Comment:
I don't think these should exist in this file.

I think they should only be exported in `index.native.js` and if another `.native.js` file needs these it would import from `common-adapters/index.native`.

I realize it's a bit more work than just importing everything from `common-adapters` but I think things that are exported in `commond-adapters` here should be common to both desktop and native.

Also these will have types that only make sense on Native, so putting these in a common type file feels hacky.